dockerfile

```diff
@@ -0,0 +1,38 @@
+# build stage
+## import image
+FROM golang:alpine AS build_base
+
+## add git
+RUN apk add --no-cache git
```

Comment:
I am uncertain why Git is needed in the Dockerfile. Could you explain your motive?